refarch-integrations/refarch-email-integration/refarch-email-integration-rest-example/src/main/resources/application-local.yml

```diff
@@ -6,6 +6,25 @@ spring:
     port: 1025
     username: test@muenchen.de
     password: secret
+  security:
+    oauth2:
+      client:
+        provider:
+          keycloak:
+            issuer-uri: http://keycloak:8100/auth/realms/local_realm
+            user-info-uri: ${spring.security.oauth2.client.provider.keycloak.issuer-uri}/protocol/openid-connect/userinfo
+            jwk-set-uri: ${spring.security.oauth2.client.provider.keycloak.issuer-uri}/protocol/openid-connect/certs
+            # used for RequestResponseLoggingFilter in s3-rest-service
+            user-name-attribute: user_name
```

Comment:
The RequestResponseLoggingFilter and the corresponding  property `user-name-attribute` are LHM specific. Both should be removed from the integration lib.

Reply:
Removed as not needed by test.